### PR TITLE
People are now hidden behind smoke

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Smoke now appears in front of people, you can click through it, and users with thermals or xray can see through smoke to see the people in the smoke.

https://github.com/ParadiseSS13/Paradise/assets/91113370/708b808a-ce76-4376-babc-790c06d7223d

This PR also adds a admin outfit for a TSF infiltrator outfit that is DEFINITELY not solid snake, but is not the major focus of this PR, its just something cool I made to show off this PR, and then wanted to add it to the full game.

I also fixed a weird bug where nodrop belts could get stuck to your hand if you drag them into your hand.

image

## Why It's Good For The Game
I thought it was weird that people appear in front of smoke, and wanted to make it proper so that people appear behind them. Then I got a little carried away. I can remove the rest if needed.

## Images of changes

Thermal vision:
image

X-ray vision:
image

## Testing

See the videos and images above

## Changelog
:cl:
tweak: Mobs now hidden behind smoke, instead of in front of it. Players with thermals or x-ray will be able to see through the smoke.
fix: You can no longer get nodrop belts stuck in your hand by dragging them from your belt-slot to your hand.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -648,6 +639,15 @@
 	for(var/I in 1 to 7)
 		new /obj/item/grenade/flashbang(src)
 
+/obj/item/storage/box/smoke_grenades
+	name = "smoke grenades"
+	desc = "A box with 7 snoke grenades."
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/boxes.dm:644]:
```suggestion
	desc = "A box with 7 smoke grenades."
```
Please tell me your intent here was snaaaaaake grenades.

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -348,6 +348,9 @@
 /obj/item/radio/headset/ert/alt/solgov
 	name = "\improper Trans-Solar Federation Marine's bowman headset"
 
+/obj/item/radio/headset/ert/alt/solgovviper
+	name = "\improper Trans-Solar Federation Infilitrator's bowman headset"
```
Comment (Henri215) [inline code/game/objects/items/devices/radio/headset.dm:352]:
```suggestion
	name = "\improper Trans-Solar Federation Infiltrator's bowman headset"
```

File: code/modules/clothing/under/solgov.dm [hunk 3]
```diff
@@ -36,3 +36,8 @@
 	desc = "A formal uniform worn by the diplomatic representatives of the Trans-Solar Federation."
 	icon_state = "solgovr"
 	item_color = "solgovr"
+
+/obj/item/clothing/under/solgov/viper
+	name = "\improper Trans-Solar Federation Infilitor uniform"
```
Comment (Henri215) [inline code/modules/clothing/under/solgov.dm:41]:
```suggestion
	name = "\improper Trans-Solar Federation Infiltrator uniform"
```

File: code/game/objects/items/weapons/storage/belt.dm [hunk 4]
```diff
@@ -502,6 +502,25 @@
...
+	desc = "Holds smokebombs, bolas, and knives. Excellent for sneaking around."
+	icon_state = "securitybelt"
+	item_state = "security"
+	max_w_class = WEIGHT_CLASS_NORMAL
+	max_combined_w_class = 18
+	can_hold = list(
+		/obj/item/grenade/smokebomb,
+		/obj/item/restraints/legcuffs/bola,
+		/obj/item/kitchen/knife/combat
+		)
```
Comment (lewcc) [inline code/game/objects/items/weapons/storage/belt.dm:516]:
```suggestion
	)
```

**Review by lewcc — APPROVED** [review]
LGTM

**Pull request merged**